users.json

```diff
@@ -212,7 +212,8 @@
         "frankyn",
         "jskeet",
         "quartzmo",
-        "jesselovelace"
+        "jesselovelace",
+        "virost"
```

Comment:
I think you need to use your github username.

Reply:
oh that makes sense. fixed